metrics/memperf/sysbench_memory.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
```

Comment:
2018

Reply:
well.... nope - the patches were submitted in Dec 2017 ;-)

Reply:
sure, if I need to re-write them, I'll try to remember to update the copyright